bugsnag-android-core/src/main/java/com/bugsnag/android/NativeInterface.java

```diff
@@ -234,7 +234,7 @@ public static String getNativeReportPath() {
      */
     @NonNull
     public static List<Breadcrumb> getBreadcrumbs() {
-        Queue<Breadcrumb> store = getClient().breadcrumbs.store;
+        Queue<Breadcrumb> store = getClient().breadcrumbState.getStore();
```

Comment:
Possibly my lack of Kotlin familiarity, but why call `getStore` and use the `store` field?

Reply:
Kotlin properties automatically synthesise getters + setters. So when called from Java the `store` property actually looks like this behind the scenes: 

```
private final Map<String, Object> store;

public Map<String, Any> getStore() {
    return store;
}
```

Therefore anywhere that previously used a field has been updated to use the getter, as the field is not accessible.